saleor/order/models.py

```diff
@@ -64,6 +64,12 @@ class Order(models.Model, ItemSet):
                                              editable=False)
     token = models.CharField(
         pgettext_lazy('Order field', 'token'), max_length=36, unique=True)
+    currency = models.CharField(
+        pgettext_lazy('Order field', 'currency'),
+        max_length=3, default=settings.DEFAULT_CURRENCY)
+    rate = models.DecimalField(
```

Comment:
Agreed, while we may need to know the currency and rate in future (to correctly show the order to the customer reviewing past orders), we should not need it for Wombat. We only need a total in shop's default currency.